Zend/zend_language_scanner.l

```diff
@@ -1613,6 +1613,11 @@ OPTIONAL_WHITESPACE_OR_COMMENTS ({WHITESPACE}|{MULTI_LINE_COMMENT}|{SINGLE_LINE_
 	RETURN_TOKEN_WITH_IDENT(T_NEW);
 }
 
+<ST_IN_SCRIPTING>"clone"{OPTIONAL_WHITESPACE_OR_COMMENTS}"(" {
+	yyless(5);
+	RETURN_TOKEN_WITH_STR(T_NAME_FULLY_QUALIFIED, 0);
```

Comment:
Would lexing it as T_STRING instead work?

Reply:
That would work, but “require” a special case in the compiler to treat `clone($x, $y)` in a namespace as a fully-qualified call.